testnet/stacks-node/src/node.rs

```diff
@@ -742,6 +768,57 @@ impl Node {
                     if blocks.len() == 0 {
                         break;
                     } else {
+                        for block in blocks.iter() {
```

Comment:
Can this be its own method?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2453/commits/4c208faa2bdc33dee45ed0e6519c8a2c94ad415e